.github/workflows/tests_lqcpu_python.yml

```
@@ -247,14 +249,41 @@ jobs:
   upload-to-codecov-linux-python:
     if: github.event_name == 'pull_request'
     needs: [test_lightning_qubit_wheels]
+    strategy:
+      matrix:
+        pl_backend: ["lightning_qubit"]
+        blas: ["OFF"]
```

Don't we need to report coverages for blas=ON? :thinking: 
```suggestion
        blas: ["OFF", "ON"]
```

---

this is just so that the source files that come with the wheels are present. The `pennylane_lightning` source files with and without blas should be the same?

---

Updated so we don't install the wheel again, just combine them by specifying the source path in .coveragerc